Sources/ArcGISToolkit/Components/UtilityNetworkTrace/UtilityNetworkTraceViewModelTrace.swift

```diff
@@ -82,19 +82,21 @@ extension UtilityNetworkTraceViewModel {
         
         /// The extent of the trace's geometry result with a small added buffer.
         var resultExtent: Envelope? {
-            if let resultEnvelope = GeometryEngine.combineExtents(of: [
+            let geometries = [
                 utilityGeometryTraceResult?.multipoint,
                 utilityGeometryTraceResult?.polygon,
                 utilityGeometryTraceResult?.polyline
```

Comment:
Instead of unwrapping `utilityGeometryTraceResult` three different times, it would be good to guard that it is not nil.

```suggestion
            guard let utilityGeometryTraceResult else { return nil }
            
            let geometries = [
                utilityGeometryTraceResult.multipoint,
                utilityGeometryTraceResult.polygon,
                utilityGeometryTraceResult.polyline
```